sta_exercise_final.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # 1. merge the stimsr and spiksr for each trial
</pre><p>This can simply be done with histogram in one line/command (use np.histogram on FFspksR directly). See if you can figure out how....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NeuroTheoryUMD/mattjac/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>This makes sense. With the simpler design matrix, this is easier to do.</p>